modules/virt-networking-glossary.adoc

```diff
@@ -20,6 +20,10 @@ API resource that allows you to define custom resources, or an object defined by
 
 Network attachment definition (NAD):: A CRD introduced by the Multus project that allows you to attach pods, virtual machines, and virtual machine instances to one or more networks.
 
+UserDefinedNetwork (UDN):: A namespace-scoped CRD introduced by the user-defined network API that tenant owners can use to create a network that isolates their namespace from other namespaces.
```

Comment:
Is this true on 4.18 ? Don't we require cluster-admin to actually create the namespace ?

Reply:
The wording in the [main doc](https://docs.openshift.com/container-platform/4.18/virt/vm_networking/virt-connecting-vm-to-primary-udn.html) is also similar to the above statement.

Reply:
Oh I missed that. Miguel is right, this statement is misleading. We should drop the mention of tenant owners. Just say that it can be used to create tenant networks

Reply:
Okay. Then, we'll need to change the wording [here](https://docs.openshift.com/container-platform/4.18/virt/vm_networking/virt-connecting-vm-to-primary-udn.html) as well.

Reply:
@phoracek Can you please review again?